navit/android/src/org/navitproject/navit/Navit.java

```diff
@@ -83,7 +83,7 @@
     private static final int           NavitDownloaderSelectMap_id     = 967;
     private static final int           NavitAddressSearch_id           = 70;
     private static final int           NavitSelectStorage_id           = 43;
-    private static final String        NAVIT_PACKAGE_NAME              = "org.navitproject.navit";
+    public static final String         NAVIT_PACKAGE_NAME              = "org.navitproject.navit";
```

Comment:
Hi,
Can you tell me a good reason for this ?
and why public ?
thx

Reply:
Hi. I made it public because I'm using it in NavitGraphics to filter out Navit's package name from the intent list, but maybe there is a better way...
https://github.com/navit-gps/navit/blob/bf282c648b3f6648405f00a69489006ab909f9d1/navit/android/src/org/navitproject/navit/NavitGraphics.java#L242

Reply:
Hi,
public is only for interaction with the world outside of Navit, so even if you would preserve this solution, package private is more than enough. 
Almost each time we have smth. with public it is some method Android will call, like the onTouch and so. 

Then the subject itself, if you have a getter for smth. then it is preffered to use that as you did before by using getPackageName(). 
I was attracted to take a look into this commit because of the word 'guesswork' in the title of the commit, did the solution with the getter give you strange results ?

I plan to remove the hardcoded packagename in Navit.java on the first occasion, I have already done quite a bit of cleanup and that is a leftover.
Where does it all lead to ? the coupling between the classes gets lower

Does your work to filter out Navit from the list work ok ?

THX!

Reply:
OK, I reverted back to the previous solution: it did work, but when I saw the package name value was already directly accessible as a string, I switch to that solution.

Filtering out navit from the list does work (tested on 4 devices). Please have a look, it should work in all cases (no app for intent, 1 app for intent and multiple apps for intent), without proposing navit and thus without any crash anymore.
If so, this PR will be ready for merge.